kpops/component_handlers/helm_wrapper/helm.py

```diff
@@ -198,14 +198,11 @@ def get_manifest(self, release_name: str, namespace: str) -> Iterable[HelmTempla
     def get_version(self) -> Version:
         command = ["helm", "version", "--short"]
         short_version = self.__execute(command)
-        version_match = re.search(r"^v(\d+(\.\d+){0,2})", short_version)
+        version_match = re.search(r"^v(\d+(?:\.\d+){0,2})", short_version)
         if version_match is None:
             raise RuntimeError("Could not parse the Helm version.")
```

Comment:
should we add the output of Helm to the error message?

Reply:
sounds useful, I would do it

Reply:
wdyt?

Reply:
looks good